### DISCUSSION TITLE
DB Backend is RLOG, but an incompatible OTP version has been detected

### DISCUSSION THREAD

**Discussion by capedra** [emqx/emqx#8592] [Q&A] [answered]
Hello. I've been getting the following error whenever I start the "EMQX 5.0.4-g390ebd8f" that I compiled using OTP 24.3.3:
```
DB Backend is RLOG, but an incompatible OTP version has been detected. Falling back to using Mnesia DB backend.
```
So, what's going on? And what's the exact version that I should use to compile the latest EMQX from Github? Thanks !!

**Comment by ieQu1** [comment, marked as answer]
Hello,

This error message is mostly harmless, unless you want to use of EMQX replicant nodes: https://www.emqx.io/docs/en/v5.0/deploy/cluster/db.html#node-roles
We use our own fork of OTP that, among other things, contains a patch needed for this feature to work: https://github.com/emqx/otp 
EMQX 5.0 is currently using `OTP-24.1.1-emqx-1` tag.

You can still use the upstream OTP if you wish, but to silence this message you need to set `node.db_backend` configuration parameter to `mnesia`.

**Reply by camima2022** [reply]
i have the same question . it's an error, i can't run "_build/emqx/rel/emqx/bin/emqx start
"

**Reply by SergeTupchiy** [reply]
Hi @camima2022, 
Please share more details, is it the only error you see? 
Please also check and share logs in `_build/emqx/rel/emqx/log` and OS you are using.
